### DISCUSSION TITLE
Failed to rollback db to original column type.

### DISCUSSION THREAD

**Discussion by FranciscoCaldeira** [laravel/framework#41140] [Q&A]
- Laravel Version: 8.77.1
- PHP Version: 7.4.3
- Database Driver & Version: mysql & innodb_version 10.5.5

### Description:
I have started the column with tinyText(), changed to string(), and the rollback to tinyText(), but can't rollback to tinyText(), because causes an `Doctrine\DBAL\Exception`.

> Unknown column type "tinytext" requested. Any Doctrine type that you use has to be registered with \Doctrine\DBAL\Types\Type::addType(). You can get a list of all the known types with \Doctrine\DBAL\Types\Type::getTypesMap(). If this error occurs during database introspection then you might have forgotten to register all database types for a Doctrine Type. Use AbstractPlatform#registerDoctrineTypeMapping() or have your custom types implement Type#getMappedDatabaseTypes(). If the type name is empty you might have a problem with the cache or forgot some mapping information.

### Steps To Reproduce:
Create a migration that on up() has a column with tinyText(), change it with `change( )` to the type to string(), and
`down()` method has a [tinyText()](https://laravel.com/docs/9.x/migrations#column-method-tinyText). Do a `php artisan migrate:rollback`

**Comment by hafezdivandari** [comment]
This is already fixed at #44101
